### ISSUE TITLE
Why Batch Norm Causes Exploding Gradients

### ISSUE THREAD

**Issue by masteusz** [masteusz/projects-tracker#19]
#### Article
https://kyleluther.github.io/2020/02/18/batchnorm-exploding-gradients.html
https://openreview.net/pdf?id=SyMDXnCcF7

## Key Takeaways
* 
* 

## Useful Code Snippets
```python
#Add your codes here
import numpy as np
import pandas as pd

```

## Useful Tools
* 
*
